docs/src/components/page-shell.js

```diff
@@ -4,6 +4,10 @@ import PageLayout from '../../../src/components/page-layout';
 
 import navigation from '@mapbox/batfish/data/navigation'; // eslint-disable-line
 import topics from '@mapbox/batfish/data/topics'; // eslint-disable-line
+import level from '@mapbox/batfish/data/level'; // eslint-disable-line
+import language from '@mapbox/batfish/data/language'; // eslint-disable-line
+import products from '@mapbox/batfish/data/products'; // eslint-disable-line
+import videos from '@mapbox/batfish/data/videos'; // eslint-disable-line
```

Comment:
I could imagine bundling all of these together into a larger "filterables" object, but we could leave them like this for now and follow up later!

Reply:
I agree, I think we should bundle these into a single object to avoid having to have added configuration in the page-shell and and batfish.config. I'll make some adjustments to the batfish helper to help bundle them.